main.go

```diff
@@ -1070,6 +1070,10 @@ func appendConfigFromFile(config *Config, path string) error {
 	if password, ok := file.Get("ssh-tunnel", "password"); ok {
 		config.SSHConnConfig.Password = password
 	}
+
+	if keyfile, ok := file.Get("ssh-tunnel", "keyfile"); ok {
+		config.SSHConnConfig.KeyFile = keyfile
+	}
 	return nil
```

Comment:
I'm guessing this was inadvertently included from another commit?

Reply:
Ah yep was looking at also adding a key file option to support things like vagrant easier

Reply:
updated https://github.com/jackc/tern/commit/d08e1b7273f60b94c8e27e83b8abb008789acde5